modules/ROOT/pages/index.adoc

```diff
@@ -16,7 +16,7 @@ Your non-MuleSoft microservices might be written using different languages and p
 
 == Anypoint Service Mesh Architecture
 
-A service mesh is an independent architecture layer, which is encapsulated in a Kubernetes Cluster. In a service mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  
+Anypoint Service Mesh is an independent architecture layer, which is encapsulated in a Kubernetes or Red Hat OpenShit cluster. In the Anypoint Service Mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  
```

Comment:
```suggestion
Anypoint Service Mesh is an independent architecture layer, which is encapsulated in a Kubernetes or Red Hat OpenShift cluster. In the Anypoint Service Mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  
```

Reply:
Done! Thanks!